src/Ledger/GovernanceActions.lagda

```diff
@@ -145,15 +127,21 @@ module _ (pp : PParams) (ccThreshold' : Maybe ℚ) where
     vote = just
 
   threshold : GovAction → GovRole → Maybe ℚ
-  threshold NoConfidence           = λ { CC → noVote           ; DRep → vote P1     ; SPO → vote Q1 }
-  threshold (NewCommittee _ _ _)   = case ccThreshold' of λ where
-                        (just _)   → λ { CC → noVote           ; DRep → vote P2a    ; SPO → vote Q2a }
-                        nothing    → λ { CC → noVote           ; DRep → vote P2b    ; SPO → vote Q2b }
-  threshold (NewConstitution _ _)  = λ { CC → vote ccThreshold ; DRep → vote P3     ; SPO → noVote }
-  threshold (TriggerHF _)          = λ { CC → vote ccThreshold ; DRep → vote P4     ; SPO → vote Q4 }
-  threshold (ChangePParams x)      = λ { CC → vote ccThreshold ; DRep → vote (P5 x) ; SPO → noVote }
-  threshold (TreasuryWdrl _)       = λ { CC → vote ccThreshold ; DRep → vote P6     ; SPO → noVote }
-  threshold Info                   = λ { CC → vote 2ℚ          ; DRep → vote 2ℚ     ; SPO → vote 2ℚ }
+  threshold = λ where
```

Comment:
Again, the old one is better for the PDF here. Also, since this forms a nice table that exists exactly like that in the [CIP](https://github.com/WhatisRT/CIPs/blob/post-edinburgh/CIP-1694/README.md#requirements) we should keep it as much like that as possible.

Reply:
This is hidden, so probably the PDF requirement does not apply here, right?

Indeed the correspondence with the table is nice, but it leads to long lines, which us Agda developers cannot read probably (I necessarily have to zoom out to see the table without newlines) and if this is eventually included in the PDF it will overflow beyond the right edge of the page.

Thus, I don't see a way to get something better than what I did here, respecting size while keeping some faint reminder of the table.

Reply:
The fact that this is hidden is also an issue, it should be included in the PDF 😅

One thing we could do is replace the pattern-lambda that matches on `Role` by a function that just takes three arguments.

Reply:
I don't understand the fix here. Also, if this is to be revealed it has to be significantly changed because it is under a parametrized `module`. Shall I open a separate issue for this to be done properly?

Reply:
Or we could just make the function return the triple, and convert it by composing it with `fromTriple` (or whatever a good name for this function would be).